docs/release-notes/cloud/cloud-2021-10.md

```diff
@@ -0,0 +1,77 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud October 2021.
+included_jira_versions: ['2021.Q4.2', '2021.Q4.3']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/3.9.12
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/3.9.48
+---
+
+# Cloud October 2021
+
+These release notes are for the Codacy Cloud updates during October 2021.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   Added the plugin [<span class="skip-vale">@next/eslint-plugin-next</span>](https://www.npmjs.com/package/@next/eslint-plugin-next) to [<span class="skip-vale">codacy-eslint</span>](https://github.com/codacy/codacy-eslint). (CY-5071)
+
+## Bug fixes
+
+-   Added protection against CSRF attacks targeting the Codacy API v3. In the case of phishing, even if the victim opens a malicious link, the <span class="skip-vale">attack</span> won't work. CVSS v3.1 score: 6.4 (Medium) (CY-5131)
+-   Fixed an issue that prevented Codacy from displaying secondary email addresses from GitLab accounts. (CY-5130)
+-   Fixed an issue that was causing organizations using the legacy Startup plan to get wrong messages about missing members and to not be able to reanalyze commits and PRs. (CY-5112)
```

Comment:
should we mention Startup? Can't we just say "a legacy plan". Looks like we are giving importance to something that doesn't exist, idk, just a feeling.

Reply:
Good suggestion @inesgomas! It makes sense. :+1: